ansible/roles/baselayout/tasks/partials/repo/centos7.yml

```diff
@@ -11,6 +11,12 @@
     name: "https://centos{{ ansible_distribution_major_version }}.iuscommunity.org/ius-release.rpm"
     state: present
 
+- name: centos7 | install scl for devtoolset-6
+  when: "arch != 'arm64'"
```

Comment:
Will need to exclude ppcle as well, I found: https://github.com/nodejs/build/pull/1763#discussion_r275883830